gov2/sns/Subscribe/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "EmailAddress": "noone@nowhere.com",
```

Comment:
Should only use example.com/.net/.org domain per group style guide.

Reply:
Fixed